provider/ocm_policies_data_source.go

```diff
@@ -0,0 +1,194 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provider
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift-online/ocm-sdk-go/logging"
+)
+
+const (
+	// Policy IDs from type operator roles
+	CloudCred       = "openshift_cloud_credential_operator_cloud_credential_operator_iam_ro_creds_policy"
+	CloudNetwork    = "openshift_cloud_network_config_controller_cloud_credentials_policy"
+	ClusterCSI      = "openshift_cluster_csi_drivers_ebs_cloud_credentials_policy"
+	ImageRegistry   = "openshift_image_registry_installer_cloud_credentials_policy"
+	IngressOperator = "openshift_ingress_operator_cloud_credentials_policy"
+	MachineAPI      = "openshift_machine_api_aws_cloud_credentials_policy"
+
+	// Policy IDs from type account roles
+	Installer            = "sts_installer_permission_policy"
+	Support              = "sts_support_permission_policy"
+	InstanceWorker       = "sts_instance_worker_permission_policy"
+	InstanceControlPlane = "sts_instance_controlplane_permission_policy"
+)
+
+type OcmPoliciesDataSourceType struct {
+}
+
+type OcmPoliciesDataSource struct {
+	logger       logging.Logger
+	awsInquiries *cmv1.AWSInquiriesClient
+}
+
+func (t *OcmPoliciesDataSourceType) GetSchema(ctx context.Context) (result tfsdk.Schema,
+	diags diag.Diagnostics) {
+	result = tfsdk.Schema{
+		Description: "List of rosa operator role for a specific cluster.",
+		Attributes: map[string]tfsdk.Attribute{
+			"operator_role_policies": {
+				Description: "Operator role policies.",
+				Attributes:  operatorRolePoliciesNames(),
+				Computed:    true,
+			},
+			"account_role_policies": {
+				Description: "Account role policies.",
+				Attributes:  accountRolePoliciesNames(),
+				Computed:    true,
+			},
+		},
+	}
+	return
+}
+
+func accountRolePoliciesNames() tfsdk.NestedAttributes {
+	return tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+		"sts_installer_permission_policy": {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		"sts_support_permission_policy": {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		"sts_instance_worker_permission_policy": {
+			Type:     types.StringType,
+			Computed: true,
+		},
+		"sts_instance_controlplane_permission_policy": {
+			Type:     types.StringType,
+			Computed: true,
+		},
+	})
+}
+
+func operatorRolePoliciesNames() tfsdk.NestedAttributes {
+	return tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+		"openshift_cloud_credential_operator_cloud_credential_operator_iam_ro_creds_policy": {
```

Comment:
Could use the consts you setup above
https://github.com/terraform-redhat/terraform-provider-ocm/pull/36/files#diff-229c93a91ec2ce9ad63f6c459ed2a928c23d23bed45534f77ddc80387e7db576R31